CMakeLists.txt

```diff
@@ -13,6 +13,17 @@ include(helpers)
 
 find_package(Python3 COMPONENTS Interpreter)
 
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Build Options
+option(DEVELOPER_MODE "enable developer checks, treats warnings as errors" OFF)
+option(BUILD_TESTS "Build unit tests." ON)
+option(FORMAT_CPP_STYLE "format code style of C++ sources" OFF)
+option(USE_ASAN "enable AddressSanitizer" OFF)
+option(USE_UBSAN "enable UndefinedBehaviorSanitizer" OFF)
+option(USE_MSAN "enable MemorySanitizer" OFF)
```

Comment:
I feel these CMake options should have the `UR_` prefix since we integrate into other projects we should avoid using option names which might conflict with those projects. Taking LLVM as an example, all options in that project have the `LLVL_` prefix unless they are options builtin to CMake.

Reply:
Done - I added `UR_`. I removed the `${PROJECT_NAME}` prefix because it was cumbersome and wasn't being used correctly.

Reply:
Yeah, `${PROJECT_NAME}` prefix is cumbersom.